jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/operation/WebServerControlRunSteps.java

```diff
@@ -46,6 +47,26 @@ public void clickWebServerControlLink(final String linkLabel, final String webSe
                 + webServerName + "']/following-sibling::td//button[text()='" + linkLabel + "']"));
     }
 
+    @When("^I click the httpd.conf link of web server \"(.*)\" under group \"(.*)\" in the operations tab and I see httpd.conf$")
+    public void clickWebServerHttpdConfLink(final String webServerName, final String groupName) {
+        origWindowHandle = jwalaUi.getWebDriver().getWindowHandle();
+        int count = 0;
+        while(count < 3) {
```

Comment:
We should make the max number of retries a configurable property with a default value of 3